volumes/csi-wrapper/hack/ibm/cleanandreinstall.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
-# this script is for develop test only, user must prepared env via wrap-ibm-vpc-block-csi-driver.md 
+# this script is for develop test only, user must prepared env via wrap-ibm-vpc-block-csi-driver.md
 kubectl delete po nginx
 kubectl delete pvc my-pvc
 kubectl -n kube-system delete po nginx
 kubectl -n kube-system delete pvc my-pvc
 pv_name=$(kubectl get pv |grep ibmc-vpc-block-5iops-tier|awk '{print $1}')
 kubectl delete pv $pv_name
-cd ~/ibm-vpc-block-csi-driver
```

Comment:
what's the problem here?

Reply:
I mean, I wonder if exiting the script at first directory failure is the right thing. As this seems a cleanup script, perhaps it should move to the next  block if a given directory doesn't exist, to continue the clean up process.

Reply:
I added your suggestion.